mule-migration-tool-library/src/main/java/com/mulesoft/tools/migration/library/mule/steps/nocompatibility/TranslateInboundReferencesStep.java

```diff
@@ -65,36 +65,39 @@ public String getDescription() {
   }
 
   @Override
-  public void execute(ApplicationModel applicationModel, MigrationReport report) throws RuntimeException {
-    ApplicationGraph applicationGraph = applicationModel.getApplicationGraph();
-    // resolve inbound references per flow
-    // TODO: this needs to be moved into actual steps
-    applicationGraph.getAllFlowComponents().forEach(flowComponent -> {
-      //TODO: change to only find the source when we have to do a property translation in a flow component
-      Optional<PropertiesSource> propertiesEmitter = applicationGraph.findClosestPropertiesSource(flowComponent);
-      if (propertiesEmitter.isPresent()) {
-        Queue<Element> elementsToUpdate = new LinkedList<>();
-        elementsToUpdate.offer(flowComponent.getXmlElement());
-
-        while (!elementsToUpdate.isEmpty()) {
-          Element element = elementsToUpdate.poll();
-          // update references in attributes
-          updatePropertyReferencesInAttributes(element, propertiesEmitter.get().getType(), report);
-          Optional<CDATA> cdataContent = getCDATAContent(element.getContent());
-          if (element.getChildren().isEmpty() && cdataContent.isPresent()) {
-            updatePropertyReferencesInCDATAContent(element, cdataContent.get(), propertiesEmitter.get().getType(), report);
-          } else {
-            element.getChildren().forEach(e -> elementsToUpdate.offer(e));
+  public void execute(Element unused, MigrationReport report) throws RuntimeException {
```

Comment:
Not required for this task, but this step should be removed and the translation logic be part of the expression migrator

Reply:
there is a task in the sprint already to do this, I just didn't want to tackle everything in the same PR